content/articles/building-a-simple-translation-app-using-python-for-beginners/index.md

```diff
@@ -0,0 +1,171 @@
+- [Downloading the pre-trained translation model](#downloading-the-pre-trained-translation-model)
+- [Building the web app for interaction using Gradio](#building-the-web-app-for-interaction-using-gradio)
+- [Wrapping up](#wrapping-up)
+
+### Introduction
+The queries you make on websites such as Google and smart assistance such as Google assistant all have one thing in common, they both use Natural Language Process (NLP). NLP is a cutting-edge AI technology that gives a machine the ability to read, understand, and derive meaning from human languages. NLP is the reason why Google assistant sounds so intelligent and Google queries are responded to so intelligently. Data analysts and machine learning experts use NLP be mimic human linguistics behavior saving manpower and time. Every day, you use NLP unknowingly in your phone's auto-correct feature helping you correctly spell a word. To understand how NLP works in-depth, please refer to this [article](https://www.section.io/engineering-education/five-real-life-use-cases-of-natural-language-processing-nlp/).
+
+NLP is a huge library that can be difficult to understand for beginners. Because of this, easy-to-use libraries such as [Hugging Face](https://www.section.io/engineering-education/hugging-face/) have been created to democratize NLP. It is one of the leading startups in the NLP space. This tutorial aims to use it to build a simple translation app.
+
+Hugging Face is a great open-source library that is doing powerful work in the Natural Language Processing (NLP) space. The library has a bunch of pre-trained models which you can leverage or fine-tune. The library has many models including BERT and GPT-2 models that perform various tasks, but for our case, we'll be leveraging the pre-trained language pipeline. Rather than going ahead and training a huge language model such as GPT, one can leverage the ML pipeline instead.
+
+Pipelines are a great and easy way to use models for inference. Hugging Face has many pipelines including the `TokenClassificationPipeline`, `ZeroShotClassificationPipeline` and `Text2TextGenerationPipeline` just to name a few. In our tutorial, we'll be using the `TranslationPipeline`. This pipeline has been fine-tuned to perform translation tasks.
+
+Technology companies such as [Speech Brain](https://speechbrain.github.io/) and [GoogleAI](https://ai.google/research/) use the hugging face model to perform deep learning and create speech technologies.     
+
+[Gradio](https://gradio.app/), on the other hand, allows us to quickly create customizable UI components around PyTorch models. It allows you to create interfaces for your ML models. It helps you quickly test the models you've created using a web interface. There are free example code implementations on their official website demonstrating how you can use gradio to perform digit classification, face segmentation, and outbreak forecast. Please feel free to check it out.
```

Comment:
Capitalize the "g" in "gradio", and edit the last sentence to "The reader is encouraged to take a look at the examples in the documentation for better understanding."